community/index.tt

```diff
@@ -324,10 +324,6 @@
       <p>Works on improving learning experience and documentation.</p>
       <a class="button -primary" href="[%root%]community/teams/documentation.html">Read more</a>
 
-      <h3>Nixpkgs Architecture Team</h3>
-      <p>Solves architectural Nixpkgs issues that are too big in scope for any single person to undertake.</p>
-      <a class="button -primary" href="[%root%]community/teams/nixpkgs-architecture.html">Read more</a>
```

Comment:
For some reason this links to `/community/teams/nixpkgs-architecture`, without the `.html` suffix. So that's why I made the redirect for the URL without the `.html`.

Though technically the URL _with_ the `.html` suffix also serves the exact same page..